controllers/articles.controller.js

```diff
@@ -34,4 +35,20 @@ const getArticlesCommentsById = (req, res, next) => {
     .catch((err) => next(err));
 };
 
-module.exports = { getArticles, getArticlesById, getArticlesCommentsById };
+const patchArticlesById = (req, res, next) => {
+  const { article_id } = req.params;
+  const { incVote } = req.body;
+
+  updateArticles(article_id, incVote)
+    .then((votes) => {
+      res.status(200).send({ votes });
```

Comment:
needs a key of `article` - it shoulld be the updated article - with the new votes